content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -67,6 +67,10 @@ Although, Poseidon-Firmament scheduler is capable of scheduling various types of
 - Poseidon-Firmament is supported from Kubernetes release 1.6 and works with all subsequent releases.
 - Release process for Poseidon and Firmament repos are in lock step.  The current Poseidon release can be found [here](https://github.com/kubernetes-sigs/poseidon/releases) and the corresponding Firmament release can be found [here](https://github.com/Huawei-PaaS/firmament/releases).
 
+{{< caution >}}
```

Comment:
nit: Line 68. Should this page link to a third party:
https://github.com/Huawei-PaaS/firmament/releases
The Poseidon repo contains links to the Firmament page.